lib/single_test_filter.js

```diff
@@ -7,8 +7,16 @@ module.exports = function (tests, filename) {
   logger.log("Using nightwatch test filter: " + filename);
 
   return tests.filter(function (test) {
-    if (path.resolve(test.filename.trim()) === path.resolve(filename.trim())) {
-      return true;
+    if(filename instanceof Array){
```

Comment:
What was the issue with this filter? When would `fileName` be an Array?

Reply:
filename is an array when multiple files are passed in, so it was erring on filename.trim() because that method doesn't exist on an array.